app/views/search/search.html.erb

```diff
@@ -6,19 +6,15 @@
   <table class='table table-striped'>
     <thead>
       <tr>
-        <th class='font-weight-bold w-25'> News Release Number </th>
-        <th class='font-weight-bold w-25'> News Release Title </th>
-        <th class='font-weight-bold w-25'> Agency </th>
-        <th class='font-weight-bold w-25'> Region </th>
+        <th class='font-weight-bold w-50'> News Release Number </th>
+        <th class='font-weight-bold w-50'> News Release Title </th>
       </tr>
     </thead>
     <tbody>
       <% @sorted_results.each do |result| %>
         <tr>
           <td><%= link_to "#{result.news_release_number}", news_log_path(result) %></td>
           <td><%= link_to "#{result.title}", news_log_path(result) %></td>
-          <td><%= link_to "#{result.agency.name}", news_log_path(result) %></td>
-          <td><%= link_to "#{result.region.name}", news_log_path(result) %></td>
```

Comment:
we need the agency and region .. just don't make it a link.

Reply:
removed link for agency and region